framework/test/jest/specs/unit/controller/controller.spec.js

```diff
@@ -77,9 +77,8 @@ describe('Controller Class', () => {
 				spies._setupDirectories
 			);
 			expect(spies._setupBus).toHaveBeenCalledAfter(spies._validatePidFile);
-			expect(spies._loadModules)
-				.toHaveBeenCalledAfter(spies._setupBus)
-				.toHaveBeenCalledWith(modules);
+			expect(spies._loadModules).toHaveBeenCalledAfter(spies._setupBus);
+			expect(spies._loadModules).toHaveBeenCalledWith(modules);
```

Comment:
@limiaspasdaniel this was solved with https://github.com/LiskHQ/lisk/pull/3179. Could you rebase your branch to development.

Reply:
I am not really sure of rebasing this branch to development. @lsilvs can it be done in this branch?

Reply:
We will do before merging `feature/child_process_module` to development